serving/docs/lmi/user_guides/embedding-user-guide.md

```diff
@@ -0,0 +1,145 @@
+# LMI Text Embedding User Guide
+
+Text Embedding refers to the process of converting text data into numerical vectors. These embeddings capture the semantic meaning of the text and can be used for various tasks such as semantic search and similarity detection.
+
+The inference process involves:
+
+1. **Loading a Model**: Loading a model from local directory, S3 or from huggingface repository,.
+2. **Tokenization**: Breaking down the input text into tokens that the model can understand.
+3. **Embeddings**: Passing the tokens through the model to produce embeddings. Embedding is a multi-dimension vector that could be used for RAG or general embedding search.
+
+LMI supports Text Embedding Inference with the following engines:
+
+- OnnxRuntime
+- Candle
+- TensorRT
+- Python
+
+Currently, the OnnxRuntime engine provides the best performance for text embedding in LMI. 
+
+## Supported Model Architectures
+
+The following text models are supported:
+
+- Bert (`BAAI/bge-base-en-v1.5`, `BAAI/bge-small-en-v1.5`, `BAAI/bge-large-en-v1.5`, etc.)
+
+Other embedding models are also supported, but may not be as performant as the model architectures listed above.
+
+## Quick Start Configurations
+
+You can leverage LMI Text Embedding inference using the following starter configurations:
+
+### serving.properties
```

Comment:
We should not promote this usage, we should guide user to:
1. Use djl:// model if the model already exist
2. use HF_MODEL_ID to convert the model at runtime
3. Manually import the model into DJL model format with our djl_convert/djl_import tool
4. and finally fully customize with `serving.properties`

Reply:
If the user give HF_MODEL_ID, then the code will automatically convert for them.